weasyprint/formatting_structure/build.py

```diff
@@ -311,9 +319,11 @@ def marker_to_box(element, state, parent_style, style_for, get_image_from_uri,
         # See https://drafts.csswg.org/css-pseudo-4/#marker-pseudo
         marker_box.style['position'] = 'absolute'
         if parent_style['direction'] == 'ltr':
+            marker_box.style['left'] = 'auto'
             translate_x = properties.Dimension(-100, '%')
         else:
-            translate_x = properties.Dimension(100, '%')
+            marker_box.style['right'] = properties.Dimension(0, 'px')
+            translate_x = properties.Dimension(20, '%')
```

Comment:
This 20% value is a bit strange… Why doesn’t the 100% translation work?

Reply:
I am not sure why 100% did not work. I know 20 is really strange and i will investigate more.